pkg/controllers/managementuser/nodesyncer/cordonfieldssyncer.go

```diff
@@ -236,6 +236,10 @@ func (d *nodeDrain) getKubeConfig() (*clientcmdapi.Config, error) {
 }
 
 func getFlags(input *v32.NodeDrainInput) []string {
+	defaultIgnoreDaemonSets := true
+	if input.IgnoreDaemonSets == nil {
+		input.IgnoreDaemonSets = &defaultIgnoreDaemonSets
```

Comment:
youre setting this, if someone passes in nil on purpose it'll come back as true. perhaps my attempt at this in another PR is a better idea? https://github.com/rancher/rancher/blob/master/pkg/controllers/managementuser/noderemove/nodes.go#L30-L40

Reply:
yep makes sense, updated